tmt/utils/structured_field.py

```diff
@@ -0,0 +1,468 @@
+import re
+from collections import OrderedDict
+from collections.abc import Iterator
+from typing import Any, Optional, Union
+
+import fmf.utils
+
+from tmt.utils import StructuredFieldError, format_value, pure_ascii
+
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#  StructuredField
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+# TODO: deprecate ASAP
```

Comment:
I take it this is new TODO - maybe extend the comment with why it's being used?

Reply:
Yeah, that's a leftover: I tried to get this new file covered with pyright, but there were so many changes that it turned exactly into a mix of moving files and making changes to them. So I undid all my changes - at least I thought so, this one slipped through my review.

Reply:
Dropped in 2666b278